web/app/components/header/search.ts

```diff
@@ -69,21 +164,64 @@ export default class Search extends Component {
 
       this.query = input.value;
 
-      if (this.query) {
-        const params = {
-          hitsPerPage: 5,
-        };
-        const response = await this.algolia.search.perform(this.query, params);
+      if (this.query.length) {
+        this.searchInputIsEmpty = false;
+
+        try {
+          const productSearch = this.algolia.searchIndex.perform(
+            this.configSvc.config.algolia_docs_index_name + "_product_areas",
```

Comment:
Where did this index come from?

Reply:
I had to create a duplicate index to scope the searchable fields to `product`. Similar to how we have duplicate `createdTime_asc`/`createdTime_desc` indexes with their own ranking criteria.

I flagged it because I'm not sure of the backend / OSS implications, but I imagine you do.